featurebyte/api/aggregator/forward_aggregator.py

```diff
@@ -0,0 +1,176 @@
+"""
+Forward aggregator module
+"""
+from __future__ import annotations
+
+from typing import Any, List, Optional, Type
+
+from featurebyte import ChangeView, EventView, ItemView
+from featurebyte.api.aggregator.base_aggregator import BaseAggregator
+from featurebyte.api.target import Target
+from featurebyte.api.view import View
+from featurebyte.common.model_util import parse_duration_string
+from featurebyte.query_graph.enum import NodeOutputType, NodeType
+
+
+class ForwardAggregator(BaseAggregator):
+    """
+    ForwardAggregator implements the forward_aggregate method for GroupBy
+    """
+
+    @property
+    def supported_views(self) -> List[Type[View]]:
+        return [EventView, ItemView, ChangeView]
+
+    @property
+    def aggregation_method_name(self) -> str:
+        return "forward_aggregate"
+
+    def forward_aggregate(
+        self,
+        value_column: str,
+        method: Optional[str] = None,
+        horizon: Optional[str] = None,
+        blind_spot: Optional[str] = None,
+        target_name: Optional[str] = None,
+    ) -> Target:
+        """
+        Aggregate given value_column for each group specified in keys over a list of time windows.
```

Comment:
This should be a single window. Would it be clearer to mention horizon instead of window in the docstring?